evm/interchainTokenFactory.js

```diff
@@ -207,6 +209,43 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'registerGatewayTokens': {
+            let { assetApi, tokenInfoApi, batchSize } = options;
+
+            assetApi = assetApi || `${config.axelar.lcd}/axelar/nexus/v1beta1/assets/`;
+            tokenInfoApi = tokenInfoApi || `${config.axelar.lcd}/axelar/evm/v1beta1/token_info/`;
+            batchSize = batchSize || 10;
+
+            validateParameters({ isString: { assetApi, tokenInfoApi }, isNumber: { batchSize } });
+
+            const { assets } = (await axios.get(`${assetApi}${chain.name}`)).data;
```

Comment:
we have a httpGet helper. see how we query axelar auth info in gateway deployment